app/routes/message.js

```diff
@@ -0,0 +1,9 @@
+module.exports = (app) => {
+    let messageController = require('../controllers/messageController');
+
+    /**
+     * Chat creating
+     */
+    app.post('/user/messages', messageController.createChat);
```

Comment:
@adilet-reactjs Not /user
/user means that we are using user entity, in this case it's message entity